www/pfSense-pkg-squid/files/usr/local/pkg/squid.inc

```diff
@@ -1140,7 +1140,11 @@ function squid_resync_general() {
 					if (is_dir(SQUID_SSL_DB)) {
 						mwexec("/bin/rm -rf " . SQUID_SSL_DB);
 					}
-					mwexec(SQUID_LOCALBASE . "/libexec/squid/ssl_crtd -c -s " . SQUID_SSL_DB);
+					if (pfs_version_compare(false, 2.4, $g['product_version'])) {
```

Comment:
This is not a good test, as 2.4.5 will likely have squid 4 due to changes in the ports tree. It will need to test the actual squid package version or test for another files that is only present on squid 4.x.

Reply:
now it check security_file_certgen file (squid 4 only)